include/inexor/vulkan-renderer/imgui/imgui.hpp

```diff
@@ -0,0 +1,101 @@
+#pragma once
+
+#include "inexor/vulkan-renderer/wrapper/command_buffer.hpp"
+#include "inexor/vulkan-renderer/wrapper/command_pool.hpp"
+#include "inexor/vulkan-renderer/wrapper/device.hpp"
+#include "inexor/vulkan-renderer/wrapper/framebuffer.hpp"
+#include "inexor/vulkan-renderer/wrapper/graphics_pipeline.hpp"
+#include "inexor/vulkan-renderer/wrapper/make_info.hpp"
+#include "inexor/vulkan-renderer/wrapper/mesh_buffer.hpp"
+#include "inexor/vulkan-renderer/wrapper/renderpass.hpp"
+#include "inexor/vulkan-renderer/wrapper/resource_descriptor.hpp"
+#include "inexor/vulkan-renderer/wrapper/shader.hpp"
+#include "inexor/vulkan-renderer/wrapper/swapchain.hpp"
+#include "inexor/vulkan-renderer/wrapper/texture.hpp"
+
+#include <glm/glm.hpp>
+#include <imgui.h>
+#include <spdlog/spdlog.h>
+#include <vma/vk_mem_alloc.h>
+#include <vulkan/vulkan.h>
+
+#include <assert.h>
+#include <iomanip>
+#include <memory>
+#include <sstream>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <vector>
+
+namespace inexor::vulkan_renderer::imgui {
+class ImGUIOverlay {
+    const wrapper::Device &m_device;
+    const wrapper::Swapchain &m_swapchain;
+    const VkPhysicalDevice m_graphics_card;
```

Comment:
This member variable has been removed since it is already contained in `wrapper::Device`.
This is part of the bigger picture of issue https://github.com/inexorgame/vulkan-renderer/issues/238
Done.